scalastyle-config.xml

```diff
@@ -240,6 +240,18 @@ This file is divided into 3 sections:
     ]]></customMessage>
   </check>
 
+  <check customId="notthrowoutofmemory" level="error" class="org.scalastyle.file.RegexChecker" enabled="true">
+    <parameters><parameter name="regex">throw new OutOfMemoryError</parameter></parameters>
+    <customMessage><![CDATA[
+      Are you sure that you want to throw OutOfMemoryError? It will kill the hole executor, you should use
```

Comment:
hole -> whole or entire

Reply:
perhasp "it will kill the whole executor" is a bit hard to understand? (I mean, is there a way to kill part of the executor?)

Reply:
yea, maybe just kill the executor or kill the current executor.

Reply:
This rule wouldn't be specific to executor code though. Just say it's bad practice to throw Error

Reply:
Rewrite this rule for banning all throw Error in 6e49bb8.